### DISCUSSION TITLE
How to solve this problem : 'Google Api Error: notFound: Package not found: com.flexboni.flutter_fastlane. - Package not found: com.flexboni.flutter_fastlane.'

### DISCUSSION THREAD

**Discussion by flexboni** [fastlane/fastlane#18019] [Q&A]
i am using Flutter framework for my dev app.
so i adjust Fastlane in my app, but when i `fastlane supply init`, it throw error.
please give me some tips for it.

## flutter --version

![image](https://user-images.githubusercontent.com/29271126/105261135-6e7ede00-5bd2-11eb-9790-b4b037810b8a.png)

## flutter doctor -v

![image](https://user-images.githubusercontent.com/29271126/105261152-78a0dc80-5bd2-11eb-958d-31d9298e0188.png)

## fastlane -v

![image](https://user-images.githubusercontent.com/29271126/105261175-80608100-5bd2-11eb-8fe4-41000a348a43.png)

## fastlane supply init <==== ERORR!!!!!!!!!!!!!

![image](https://user-images.githubusercontent.com/29271126/105261198-8b1b1600-5bd2-11eb-8c3c-bb400c1ce8e5.png)

## fastlane/Appfile

![image](https://user-images.githubusercontent.com/29271126/105261261-aede5c00-5bd2-11eb-874a-a70920ddfc9c.png)

## package names

![image](https://user-images.githubusercontent.com/29271126/105261229-9b32f580-5bd2-11eb-86e0-393168c425b2.png)

![image](https://user-images.githubusercontent.com/29271126/105261235-9ec67c80-5bd2-11eb-9a8c-58cb4307cf0c.png)

![image](https://user-images.githubusercontent.com/29271126/105261281-b6056a00-5bd2-11eb-93b2-3280f04e8a09.png)

**Comment by Thangaraj-M** [comment]
We can use the Fastlane supply command for the apps which already present in the google play console. If the app with the given package name not present in the corresponding play console account, then it will return this error.
Please create an app in the play console and upload the apk file which generated with the same package name in any one of the track and then try the supply command.
